index.html

```diff
@@ -5096,8 +5265,36 @@ <h2 id="MediaElementAudioSourceNode">
           This node has no <a>tail-time</a> reference.
         </p>
         <dl title=
-        "[Constructor(&lt;a&gt;AudioContext&lt;/a&gt; context, &lt;a&gt;MediaElementAudioSourceOptions&lt;/a&gt; options)]interface MediaElementAudioSourceNode : &lt;a&gt;AudioNode&lt;/a&gt;"
-        class="idl"></dl>
+        "interface MediaElementAudioSourceNode : &lt;a&gt;AudioNode&lt;/a&gt;"
+        class="idl">
+          <dt>
+            Constructor
+          </dt>
+          <dd>
+            <p>
+              Let <var>node</var> be a new <a>MediaElementAudioSOurceNode</a>
+              object. <a href="#audionode-constructor-init">Initialize the
+              common part</a> of <var>node</var>, and return <var>node</var>.
+            </p>
+            <dl class="parameters">
+              <dt>
+                BaseAudioContext context
+              </dt>
+              <dd>
+                The <a>BaseAudioContext</a> this new
+                <a>MediaElementAudioSourceNode</a> will be <a href=
+                "#associated">associated</a> with.
+              </dd>
+              <dt>
+                optional MediaElementAudioSourceOptions options
+              </dt>
+              <dd>
+                Optional initial parameter value for this
+                <a>MediaELementAudioSourceNode</a>.
+              </dd>
+            </dl>
+          </dd>
+        </dl>
```

Comment:
If you haven't already, don't bother. Recall that we're going to reorder all the sections anyway to roughly alphabetical.